src/Collections-Unordered/PluggableSet.class.st

```diff
@@ -45,11 +45,17 @@ Class {
 	#category : #'Collections-Unordered-Sets'
 }
 
-{ #category : #accessing }
+{ #category : #'instance creation' }
 PluggableSet class >> integerSet [
 	^self new hashBlock: [:integer | integer hash \\ 1064164 * 1009]
 ]
 
+{ #category : #'instance creation' }
+PluggableSet class >> literalSet [
+	"A Set using equality as defined for the Literals in CompiledMethods"
+	^self new equalBlock: [ :a :b | a literalEqual: b]
```

Comment:
Are we sure the default hash it's Ok for every object answering true to `literalEqual:` ? For example `Association` implementation uses `==` but the default hash only considers the key in the association.

Reply:
I checked OCLiteralSet and it uses the default hash... which had the same problem... and I asked myself the same...